unlock-app/src/components/interface/verification/Key.tsx

```diff
@@ -169,7 +169,7 @@ export const ValidKey = ({
   const [loading, setLoading] = useState(true)
   const [viewerIsVerifier, setViewerIsVerifier] = useState(false)
   const [keyData, setKeyData] = useState({})
-  const { getKeyData, markAsCheckedIn } = useLock(lock, network)
+  const { getKeyData } = useLock(lock, network)
```

Comment:
Can you clean up the hook if this method is not used please?

Reply:
done! :)